src/utils.ts

```diff
@@ -134,6 +135,16 @@ const validateConfig = (config: JestPlaywrightConfig) => {
   }
 }
 
+export function getBrowserOptions<T>(
+  browserName: BrowserType,
+  options?: Options<T>,
+): T {
+  if (options && options[browserName]) {
+    return { ...options, ...options[browserName] }
```

Comment:
I would spread out the browserName, since its still inside right?

Will take a more detailed look later. 👍

Reply:
Yeah! That’s what I wanted. But I'm not sure if I done it nice)